examples/index.md

```diff
@@ -126,3 +126,119 @@ Content-Type: application/vnd.api+json
 for readability. In practice, these characters must be percent-encoded, as
 noted in the base specification.
 
+## Error Objects
+
+Examples of how [example objects](http://jsonapi.org/format/#error-objects) work.
+
+### Responses to failed resource creation:
+
+Response with one error object:
+
+```http
+HTTP/1.1 422 Unprocessable Entity
+Content-Type: application/vnd.api+json
+
+{
+  "errors": [
+    {
+      "source": { "pointer": "data/attributes/first_name" },
+      "detail": "is too short"
+    }
+  ]
+}
+```
+
+Response with status:
+
+```http
+HTTP/1.1 422 Unprocessable Entity
+Content-Type: application/vnd.api+json
+
+{
+  "errors": [
+    {
+      "source": { "pointer": "data/attributes/first_name" },
+      "detail": "is too short"
+    },
+    {
+      "status": "422",
```

Comment:
Great, thanks!  I made it this way on purpose since I didn't understand when error objects might have different status attributes.